pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -27,8 +27,8 @@ clusterInfo:
   consoleBaseAddress: https://` + host + `
   consoleBasePath: ""
 customization:
-  branding: okd
-  documentationBaseURL: https://docs.okd.io/4.0/
+  branding: ` + DEFAULT_BRAND + `
```

Comment:
I dont mind this, but probably would prefer a separate test as well. The problem with the tests currently is that, when they fail, the debugging is pretty difficult as its a manual comparison between 2 separate large `JSON` blobs.

Reply:
Yeah, it is hard to investigate so we probably should create some better unit test tools to help show what specifically is failing on a deep copy.

Reply:
There is a object diff output package I've heard mentioned several times to solve this problem.  It might be sufficient.